main/templates/subsidy/create.html

```diff
@@ -3,19 +3,24 @@
 
 {% block content %}
 <div id="section-subsidy">
-  {% for message in messages %}
-  <div class="alert alert-danger alert-dismissible fade show" role="alert">
-      {{ message }}
-      <button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close"></button>
-  </div>
-  {% endfor %}
+  {% if messages %}
```

Comment:
Good change! With this you are getting a good programming practise. Avoiding null values